tests/kafkatest/tests/core/consume_bench_test.py

```diff
@@ -188,12 +220,18 @@ def test_multiple_consumers_random_group_partitions(self, metadata_quorum=quorum
         self.logger.info("TASKS: %s\n" % json.dumps(tasks, sort_keys=True, indent=2))
 
     @cluster(num_nodes=10)
-    @matrix(metadata_quorum=quorum.all_non_upgrade)
-    def test_multiple_consumers_specified_group_partitions_should_raise(self, metadata_quorum=quorum.zk):
+    @matrix(
+        metadata_quorum=quorum.all_non_upgrade,
+        use_new_coordinator=[True, False]
+    )
+    @skip_if_new_coordinator_and_zk
+    def test_multiple_consumers_specified_group_partitions_should_raise(self, metadata_quorum=quorum.zk, use_new_coordinator=False):
         """
         Runs multiple consumers in the same group to read messages from specific partitions.
         It is an invalid configuration to provide a consumer group and specific partitions.
         """
+        self.kafka.use_new_coordinator = use_new_coordinator
```

Comment:
make sure in your testing this actually works by sshing onto all the brokers to see if its set properly.  The way its configured now, at `kafka.py` construction, the kraft brokers configs sets this to False, then at runtime, the override sets it to `use_new_coordinator` so it should work.